src/components/CommentForm.jsx

```diff
@@ -0,0 +1,37 @@
+import { useContext, useState } from "react"
+import { LoggedInUserContext } from '../contexts/LoggedInUser'
+import Comments from "./Comments"
+import { postComment } from "../api"
+
+const CommentForm = ({setNewComments, article_id}) => {
+    const [err, setErr] = useState(null)
+    const {loggedInUser, setLoggedInUser} = useContext(LoggedInUserContext)
+    const [commentInput, setCommentInput] = useState("")
+    const handleNewComment = (event) => {
+        event.preventDefault()
+        setErr(null)
+        setNewComments((currNewComments) => 
+            [...currNewComments, {key: currNewComments.length+1, body: commentInput, author: loggedInUser, created_at: new Date()} ]    
```

Comment:
I wouldn't optimistically render this since the created at is produced for you by posting. Instead, have an isPosting state to let the user know its being posted.

Reply:
updated this ✅  thanks so much 😄